apps/condo/domains/banking/schema/CreateBankAccountRequestService.test.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Generated by `createservice banking.CreateBankAccountRequestService '--type=mutations'`
+ */
+const faker = require('faker')
+const { makeLoggedInAdminClient, makeClient } = require('@open-condo/keystone/test.utils')
+const { expectToThrowAuthenticationErrorToResult, waitFor } = require('@open-condo/keystone/test.utils')
+const { makeClientWithProperty } = require('@condo/domains/property/utils/testSchema')
+const { createBankAccountRequestByTestClient } = require('@condo/domains/banking/utils/testSchema')
+const { Message } = require('@condo/domains/notification/utils/testSchema')
+const { MESSAGE_SENT_STATUS } = require('@condo/domains/notification/constants/constants')
+
+describe('CreateBankAccountRequestService', () => {
+    test('user: can send create bank account request if has canManageProperties access', async () => {
+        const admin = await makeLoggedInAdminClient()
+        const client = await makeClientWithProperty()
+        const payload = {
+            bankAccountClient: {
+                phone: faker.phone.phoneNumber('+7988#######'),
+                email: faker.internet.email(),
+                name: client.user.name,
+            },
+            tin: client.organization.tin,
+            name: client.organization.name,
+            propertyAddress: client.property.address,
+            organizationId: client.organization.id,
```

Comment:
What about mismatch of property address and organization properties?

Reply:
Added test for this case